contracts/Manager.sol

```diff
@@ -8,6 +8,9 @@ contract Manager is IManager {
     // Controller that contract is registered with
     IController public controller;
 
+    // Divisor used for representing percentages
+    uint256 public constant PERC_DIVISOR = 1000000;
```

Comment:
think we should add an assert anywhere in the constructor to make sure that `PERC_DIVISOR` is > 0, or even that `PERC_DIVISOR == 1000000`. It would be disasterous if it ever got changed upon deployment of a new contract, right? I guess the constructor is the only place, but I don't know if it's necessary.

Reply:
This is currently set as a constant so it wouldn't be changed in the constructor - if it is for some reason changed it would because the hardcoded constant value is changed in a newly deployed target contract. We could include asserts for `PERC_DIVISOR == 1000000` whenever we calculate percentages, but we would need to change all the assert statements as well if the value of the divisor does happen to change. 1000000 allows us to represent percentages as low as .0001% so I'm not sure if we would ever change the divisor, what do you think?

Reply:
Ok yah I don't expect us to change it intentionally. I was worried about the scenario where we deploy an updated contract and accidentally remove or add a 0. But that's probably overthinking it. I certainly don't think it needs asserts everywhere.